addons/account/models/account_move.py

```diff
@@ -4808,9 +4808,9 @@ def _add_cash_basis_lines_to_exchange_difference_vals(lines, exchange_diff_move_
         # Fix residual amounts.
         to_reconcile = _add_lines_to_exchange_difference_vals(self, exchange_diff_move_vals)
 
-        # Fix cash basis entries.
+        # Fix cash basis entries, only if not coming from the move reversal wizard.
         is_cash_basis_needed = self[0].account_internal_type in ('receivable', 'payable')
-        if is_cash_basis_needed:
+        if is_cash_basis_needed and not self._context.get('move_reverse_cancel'):
```

Comment:
I'm really note sure... When fully refunding an invoice using a credit note, you don't want any cash basis journal entry? Is it really correct?

Reply:
stable => Do not create rounding lines when coming from the wizard, the same way we don't create CABA moves
15+ => Do not add tax grids on credit note lines if the tax is cash basis

This is what was decided with @tsb-odoo